module/esami.py

```diff
@@ -52,21 +52,21 @@ def esami_cmd(userDict):
 
 	query = """SELECT anno, cdl, docenti, insegnamento{} 
 			   FROM exams
-			   WHERE 1 = 1 {} {} {}""".format(
+			   WHERE insegnamento LIKE ? {} {}""".format(
 	", " + select_sessione if select_sessione else ", prima, seconda, terza, straordinaria", #es.=> , prima, seconda, terza
 	"and not " + where_sessione + " = '[]'" if where_sessione else "", #es.=> and not prima = '[] and not terza = '[]' 
 	"and (anno = '" + where_anno + "')" if where_anno else "", #es.=>  and (anno = '1° anno' or anno = '3° anno)'
-	"and insegnamento LIKE '%" + where_insegnamento + "%'" if where_insegnamento else "" #es.=> and insegnamento LIKE '%stringa%'
```

Comment:
purtroppo python ha questo brutto modo di fare il "ternary operator", ma non ho alternative a ciò :(
ma se creassi 3 variabili anzichè fare questi tre `if else ` tutti insieme?

TIpo:
`var1 = x if a else y`
e poi  metti var1, var2 e var3

Reply:
Ad essere onesto, non mi piace molto ne com'era ne com'è ora, ma per adesso non mi viene in mente niente di meglio. Spero che almeno sia un po' più leggibile.

Reply:
ok